nvdaHelper/localWin10/oneCoreSpeech.cpp

```diff
@@ -55,104 +54,148 @@ void __stdcall ocSpeech_terminate(OcSpeech* instance) {
 }
 
 void __stdcall ocSpeech_setCallback(OcSpeech* instance, ocSpeech_Callback fn) {
-	instance->callback = fn;
+	instance->setCallback(fn);
 }
 
-void __stdcall ocSpeech_speak(OcSpeech* instance, char16 *text) {
-	String^ textStr = ref new String(text);
+void OcSpeech::setCallback(ocSpeech_Callback fn) {
+	callback = fn;
+}
+
+fire_and_forget OcSpeech::speak(hstring text) {

```

Comment:
Please add a comment here about exceptions escaping this method will result in a call to `terminate`: https://devblogs.microsoft.com/oldnewthing/20190320-00/?p=102345

Reply:
Can you still add a comment here about why we catch all exceptions.